### DISCUSSION TITLE
添加 star 收藏组件和 fab 组件拖拽

### DISCUSSION THREAD

**Discussion by KCode6** [Moonofweisheng/wot-design-uni#165] [Ideas]
### 希望fab 组件可以拖拽，如联系客客服悬浮按钮，可以在屏幕安全区（不超出屏幕，不超出 tabbar）随意拖拽
![IMG_3255](https://github.com/Moonofweisheng/wot-design-uni/assets/129662190/ed01d992-26c9-4da8-9b23-0c6bfa91f08a)

**Comment by Moonofweisheng** [comment]
[2024开发计划](https://github.com/users/Moonofweisheng/projects/1)，fab支持托拽已加入计划，“收藏小程序”欢迎pr
